apps/experiments/models.py

```diff
@@ -122,10 +123,18 @@ class ConsentForm(BaseModel):
 
     name = models.CharField(max_length=50)
     consent_text = models.TextField(help_text="Custom markdown text")
+    is_default = models.BooleanField(default=False)
```

Comment:
Should this not be unique as well?

Reply:
I could but in the future I think this will be per tenant so then it won't be unique on a different axis. I'm fine leaving it as is for now. It's a readonly field in the admin as well.

Reply:
I've also updated the model field by adding `editble=False` (https://docs.djangoproject.com/en/4.2/ref/models/fields/#editable)

Reply:
Cool!